Quarantine CylanceOPTICS Workflow with Slack (#268)
rapid7/insightconnect-workflows: workflows/Quarantine_Endpoint_with_CylanceOPTICS_from_Slack/help.md

# Description

This workflow will quarantine an endpoint in CylanceOPTICS from a Slack command. This workflow supports endpoints in the form of IP address, MAC address, hostname, or device ID.

Sample Slack Trigger Commands:

`@Rapid7 InsightConnect quarantine-endpoint 198.51.100.100`

> 

Reviewer: ```suggestion

`@Rapid7 InsightConnect quarantine-endpoint USER-DESKTOP`
```

# Description

This workflow will quarantine an endpoint in CylanceOPTICS from a Slack command. This workflow supports endpoints in the form of IP address, MAC address, hostname, or device ID.

Sample Slack Trigger Commands:

`@Rapid7 InsightConnect quarantine-endpoint 198.51.100.100`

> `@Rapid7 InsightConnect quarantine-endpoint USER-DESKTOP`

Reviewer: ```suggestion
`@Rapid7 InsightConnect quarantine-endpoint USER-DESKTOP`

```

# Description

This workflow will quarantine an endpoint in CylanceOPTICS from a Slack command. This workflow supports endpoints in the form of IP address, MAC address, hostname, or device ID.

Sample Slack Trigger Commands:

`@Rapid7 InsightConnect quarantine-endpoint 198.51.100.100`

`@Rapid7 InsightConnect quarantine-endpoint USER-DESKTOP`

> In menu from the "CylancePROTECT" page in tab "CylanceOPTICS", the quarantined endpoints will show after choose tab "Devices". When the "Device Status" is "Locked Down", endpoint is quarantined.

Reviewer: ```suggestion
When you log into CylancePROTECT, choose "CylanceOPTICS" from the left side-bar menu. The quarantined endpoints can be found on the "Devices" tab. When the "Device Status" is set to "Locked Down", the endpoint is quarantined. Note that Cylance does not support undoing the lockdown from the API, as the network access is fully removed for the endpoint.
```
